src/iidm/Identifiable.cpp

```diff
@@ -60,8 +96,23 @@ void Identifiable::extendVariantArraySize(unsigned long initVariantArraySize, un
     }
 }
 
-const std::set<std::string>& Identifiable::getAliases() const {
-    return m_aliases;
+stdcxx::const_range<std::string> Identifiable::getAliases() const {
+    return boost::range::join(m_aliasesWithoutType, m_aliasesByType | boost::adaptors::map_values);
+}
+
+stdcxx::optional<std::string> Identifiable::getAliasFromType(const std::string& aliasType) const {
+    auto it = m_aliasesByType.find(aliasType);
```

Comment:
In PR#1611, we throw an exception if aliasType is null or empty. Could you please report this change in this PR